pkg/validation/validation.go

```diff
@@ -37,3 +38,48 @@ func ParseRefWithDefault(ref string) (string, string, error) {
 	}
 	return namespace, name, err
 }
+
+var deprecatedLables = map[string]string{
+	"storageos.feature.replication":   "storageos.com/replication",
+	"storageos.feature.deduplication": "storageos.com/deduplication",
+	"storageos.feature.replicas":      "storageos.com/replicas",
+	"storageos.feature.nocompress":    "storageos.com/compression",
+	"storageos.feature.nocache":       "storageos.com/nocache", // TODO: is this one correct?
+	"storageos.feature.throttle":      "storageos.com/throttle",
+	"storageos.hint.master":           "storageos.com/hint.master",
+	"storageos.hint.docker":           "storageos.com/hint.docker",
+	"storageos.driver":                "storageos.com/driver",
+}
+
+func labeldeprecationNotice(old, new string) string {
+	depNotice := fmt.Sprintf("the label '%s' has been deprecated in favour of '%s'", old, new)
+	return depNotice + ", refer to current documentation for useage information"
```

Comment:
useage => usage

I'd change to: "refer to https://docs.storageos.com for usage details"